tests/simple/memory.wast.draft

```diff
@@ -0,0 +1,58 @@
+
+memory 0
+#assertMemory 0 pow16 .Map "Basic sanity"
+
+
+memory 0 0
+#assertMemory 0 0 .Map "Basic sanity"
+
+
+(memory 1)
+(data (i32.const 0) "WASM")
+#assertMemory 1 pow16 #asMapBytes(String2Bytes("WASM")) "Bytes"
+
+(memory 1)
+(data (i32.const 0) "WASM")
+(i32.load8_u (i32.const 0))
+#assertTopStack < i32 > 87 "W in bytes" 
+#assertMemory 1 pow16 #asMapBytes(String2Bytes("WASM")) "Byteasds"
+
+(;
+memory 0 0
+memory.size
+#assertTopStack < i32 > 0 "empty memory on stack"
+#assertMemory 0 0 .Map "empty memory"
+
+
+(memory 1)
+(data (i32.const 0) "WASM")
+(memory.size)
+#assertTopStack < i32 > 4 "size of stack" 
+
+
+  (memory 1)
+  (data (i32.const 0) "WASM")
+  (i32.load8_u (i32.const 1))
+  #assertTopStack < i32 > 65 "A in bytes" 
+
+(module
+  (memory 1)
+  (data (i32.const 0) "WASM")
+  (i32.load8_u (i32.const 2))
+  #assertTopStack < i32 > 83 "S in bytes" 
```

Comment:
Probably shouldn't have `#assert_` statements directly inside modules, as the semantics of modules will eventually not be "execute everything in the module like it was inline code anyway".

Also, we should be able to test `memory` assuming no modules exist yet. So if you want integration tests with `module`, I'd put these tests in `modules.wast`. If they are testing actual features of memories (and not just testing integration with modules), can we make them standalone memory tests instead?

Reply:
Sorry, this is from the WIP I integrated, this file is not supposed to be included.